src/vivarium_census_prl_synth_pop/components/immigration.py

```diff
@@ -130,6 +131,30 @@ def on_time_step(self, event: Event):
             event.index,
         )
 
+        num_household_immigrants = self._round_stochastically(
```

Comment:
We don't want to do this before individual immigration and then fill in the gaps when we get slightly too few household immigrants due to the last household straddling the threshold of immigrants we want? In initialization we fill that with extra GQ simulants. Are we sure we don't want to fill the gap here, especially since immigration rates are so low?

Reply:
Also, what values do we see for `self.household_immigrants_per_time_step`. If we have a value between 0 and 1, doesn't that mean we can't ever have a household immigrate with size greater than 1? We might want to draw from a different distribution that also has a mean of `self.household_immigrants_per_time_step`.